src/lib/analytics.js

```diff
@@ -47,10 +47,3 @@ document.addEventListener("click", (e) => {
 
   trackEvent(data);
 });
-
-/**
- * Configure tracking events for page changes. Emitted in `router.js`.
- */
-document.addEventListener("pageview", () => {
-  ga("send", "pageview");
```

Comment:
Doesn't removing this mean you won't get pageviews on SPA views?

Reply:
No, if you look at the PR, we used to fire a custom event which triggers this. This is now running in "loader".